spec/core_functions/color/mix.hrx

```diff
@@ -82,7 +82,7 @@ a {
 <===>
 ================================================================================
 <===> unitless_weight/firstwards/input.scss
-a {b: mix(#91e16f, #0144bf, 92)}
+a {b: mix(#91e16f, #0144bf, 92%)}
```

Comment:
Per the test this should be unitless

Reply:
I'm just going to remove these and let them be covered by the `unitless` test below.